client/state/selectors/is-jetpack-onboarding-step-completed.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { getJetpackOnboardingSettings } from 'state/selectors';
+import { JETPACK_ONBOARDING_STEPS as STEPS } from 'jetpack-onboarding/constants';
+
+export const isJetpackOnboardingStepCompleted = createSelector(
+	( state, siteId, stepName ) => {
+		const settings = getJetpackOnboardingSettings( state, siteId );
+
+		if ( ! settings ) {
+			return null;
+		}
+
+		switch ( stepName ) {
+			case STEPS.SITE_TITLE:
+				const titleModified = get( settings, 'siteTitle', '' ) !== '';
+				const defaultDescription = translate( 'Just another WordPress site' );
+				const descriptionModified =
+					get( settings, 'siteDescription', defaultDescription ) !== defaultDescription;
+				return titleModified || descriptionModified;
+			case STEPS.SITE_TYPE:
+				return !! get( settings, 'siteType' );
+			case STEPS.HOMEPAGE:
+				return !! get( settings, 'homepageFormat' );
+			case STEPS.CONTACT_FORM:
+				return !! get( settings, 'contactForm' );
+			case STEPS.BUSINESS_ADDRESS:
+				return !! get( settings, 'businessAddress' );
+			case STEPS.WOOCOMMERCE:
+				return !! get( settings, 'woocommerce' );
```

Comment:
I'm getting a false negative here, since the step is actually called `installWooCommerce`..

Reply:
Oh, that's intentional, see https://github.com/Automattic/jetpack/pull/8497/files#diff-2ccec9069213650359171727de9a2471R441

I think we should differentiate between "installing the plugin" and checking whether it's installed. But if you feel we should use the same name, I can update it.

Reply:
I think that's fine, I was just surprised 'WooCommerce' was marked incomplete even though I had clicked the corresponding button (wearing my user hat here).